cocos/base/CCEventListener.cpp

```diff
@@ -52,5 +85,15 @@ bool EventListener::checkAvailable()
     return (_onEvent != nullptr);
 }
 
+EventListener::ListenerID EventListener::getCustomListenerID(const std::string& eventName)
+{
+    static std::hash<std::string> h;
+    if (eventName.empty()) {
+        return EventListener::LISTENER_ID_CUSTOM;
+    } else {
+        return h(eventName) + EventListener::LISTENER_ID_CUSTOM;
```

Comment:
Is it possible that `EventListener::LISTENER_ID_CUSTOM` may cause overflow of `size_t`?